docs/source/UIDocumentation.md

```diff
@@ -1,160 +1,235 @@
+
+#### ```netLayout_vertical.js```
+Defines the vertical alignment of network elements. ```netLayout_vertical.js``` uses a function, called ```allocatePosition``` to give the layers their position on the canvas. It finds the closest position available to the preferred position, and checks if a node is already on that position on the X axis (if one is, a position left or right is assigned). The file also uses a custom Topolical Sort to give the nodes their position (it is based on the inputs and outputs of nodes). Later it is checked if any nodes overlap and in case they do, they are moved.
+
+#### ```netLayout.js```
+```netLayout.js``` is used to check the Y attribute of nodes. The function ```allocatePosition``` finds the closest available position to the preffered one and allocates the module. Later the nodes are checked with a DFS algorithm (depth-first search algorithm).
+
+***
+
+#### ```pane.js```
+```pane.js``` handles the layer panel and contains these methods:
+* ```toggleClass``` - toggles classes for the dropdown on the sidebar for layer selection
+* ```componentDidMount``` - handles element with layers: search for layers, shown elements, etc.
+* ```render``` - renders layer panel
+
+```pane.js``` invokes ```paneElement.js``` to render out each element.
+
+***
+
+#### ```paneElement.js```
+```paneElement.js``` renders out each element of the pane, it is invoked by ```pane.js```. It includes:
+* ```drag``` - allows dragging of layers
+* ```render``` - renders specific elements
+
+```pane.js``` renders out all of the layers for selection by the user.
+***
+#### ```panZoom.js```
+```panZoom.js``` includes the functions that zoom the canvas in and out, based on various invocations.
+
+***
+
+#### ```setParams.js```
+Contains the following methods for the layer edition sidebar:
+* ```changeProps``` - changes layer properties
+* ```changeParams``` - changes layer parameters
+* ```close``` - closes sidebar
+* ```trainOnly``` - turns model into a train-only
+* ```handleKeyPress``` - handles layer delete after the 'delete' key is pressed
+* ```componentDidMount``` - invokes ```handleKeyPress```
+* ```componentWillUnmount``` - invokes ```handleKeyPress```
+* ```render``` - renders layer options sidebar
+
+```setParams.js``` allows users to change layer parameters and reads parameters through ```data.js```
+
+***
+#### ```tabs.js```
+```tabs.js``` is used to switch between the Train and Test model buttons and to render.
+***
+#### ```tooltip.js``` and ```tooltipData.js```
 Both of these files contains code for tooltips and the ```tooltipData.js``` contains the actual tooltips that are rendered.
 
 note: tooltips are "hover" messages, they tell more information about something when a user hovers over an object.
- 
+***
+
+#### ```topbar.js```
+```topbar.js```  shows the top section of the sidebar. It has two methods:
+* ```checkURL``` - checks model import URL
+* ```render``` - renders the top section of the sidebar
```

Comment:
@kayo11 as I said before no need to mention methods for which name of function clearly specifies the function